_layouts/history.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+body_class: layout-page
+---
+
+{{ content }}
+<h1> History of the Nottingham New Theatre </h1>
```

Comment:
See above, I don't think the massive titles are strictly necessary. Going straight in with "1927 - Formation of Dramsoc" is pretty self explanatory IMO